include/dt-bindings/dac/mcp4725.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2021 Laird Connectivity
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_DAC_MCP4725_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_DAC_MCP4725_H_
+
+/* Config bits of MCP4725 DAC register are packed into top 5 bits of dac write value */
```

Comment:
Why are these defines here?  Meaning looking at the dts binding it doesn't seem like these defines would be used in the dts and as such these seem like they should be in a header in drivers/dac/dac_mcp4725.h or something like that.

Reply:
You're right, they should have belonged in drivers/dac but the whole header file has been removed in the last commit as the defines are no longer used in code.